### DISCUSSION TITLE
Anzahl von Zusatz_Ladebloecke

### DISCUSSION THREAD

**Discussion by thorsten-gehrig** [wiggal/GEN24_Ladesteuerung#64] [General]
Hi
gibt es eigentlich eine limitierung der Anzahl der Ladeblöcke?
Extrembeispiel - könnte man 12 Ladeblöcke Anlegen - für jeden Monat einen?
(Sinnhaftigkeit mal außenvor gelassen)
Gruß
Thorsten

PS: und DANKE für dieses tolle Script/Programm - was die Lebensdauer meines Akkus hoffentlich verlängert :-) I´m lovin´ it!

**Comment by wiggal** [comment]
Hi,
freut mich, wenn das Prog gefällt.
 Die Ladeblöcke sind nicht begrenzt, 12 wird kein Problem sein.
schöne Grüße 
Wiggal

**Reply by thorsten-gehrig** [reply]
Danke!
